qemu/tests/block_transfer_parameters_check.py

```diff
@@ -0,0 +1,155 @@
+"""Verify Maximum transfer length and max_sector_kb in guest """
+
+import logging
+import time
+from os.path import basename
+
+from avocado.utils import process
+from virttest import env_process
+from virttest import data_dir
+from virttest import utils_misc
+
+from virttest.iscsi import Iscsi
+from virttest.utils_misc import get_linux_drive_path
+import json
+import copy
+
+
+def run(test, params, env):
+    """
+    Verify Maximum transfer length and max_sector_kb in guest.
+
+    Steps:
+        1) Get max_sector_kb and segments of host target device.
+        2) Pass-through host target device and Boot VM.
+        3) Login guest and check the max_sector_kb and Maximum transfer length.
+        4) Execute IO Check the VM still running.(optional)
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+
+    def _verify_transfer_info(host_info, guest_info):
+        errmsg = "The guest get unexpected transfer %s" % guest_info
+        if guest_info["sectors_kb"] > host_info["sectors_kb"]:
+            test.fail(errmsg)
+        min_host = min(host_info["sectors_kb"], host_info["segments"] * 4)
+        logger.debug(min_host)
```

Comment:
also can use test.log.debug(), all look good to me.

Reply:
The logger indeed points to test.log.